ros2topic/ros2topic/api/__init__.py

```diff
@@ -57,19 +57,20 @@ def __call__(self, prefix, parsed_args, **kwargs):
 def import_message_type(topic_name, message_type):
     # TODO(dirk-thomas) this logic should come from a rosidl related package
     try:
-        package_name, *message_name = message_type.split('/')
-        if not package_name or not message_name or not all(message_name):
+        package_name, middle_module, message_name = message_type.split('/')
+        if not middle_module:
```

Comment:
This logic doesn't seem to make any sense. In which case would `not middle_module` be `True`?